arbitrum-docs/node-running/running-a-validator

```diff
@@ -0,0 +1,301 @@
+# Running a Validator
+
+Instructions for configuring a deployment for a Nitro node to be run as a validator, on either the Arbitrum One or Arbitrum Nova chain. 
+
+### Kubernetes Instructions 
+
+**Before You Begin:**
+
+- Create a PVC named `arbitrum-validator-storage` on the storage back of your choice. Examples included for AWS on GP2 EBS.
+- Create a deployment using the latest release running sleep infinity (see included sample)
+
+### Create Validator Configuration
+
+- Fill in the stubbed out values:
+    - Nova values:
+        - L1 Chain ID: 1
+        - L2 Chain ID: 42170
+    - Arb1 default values:
+        - L1 Chain ID: 1
+        - L2 Chain ID: 42161
+- `kubectl create configmap arbitrum-validator-config --from-file=config.json`
+
+### L1 Wallet Configuration:
+
+- If Using Existing Wallet:
+    - Create directory on the container: `kubectl exec <VALIDATOR_POD_NAME> -it -- bash -c "mkdir -p /home/user/data/keystore"`
+    - Copy existing wallet: `kubectl cp <EXISTING_WALLET_FILE> <VALIDATOR_POD_NAME>:/home/user/data/keystore/wallet.dat`
+- If Creating Wallet:
+    - `kubectl exec <VALIDATOR_POD_NAME> -it -- /usr/local/bin/nitro --metrics --conf.file=/config/config.json --l1.wallet.only-create-key --l1.wallet.password=<SECURE_PASSWORD>`
+
+### L1 Smart Contract Wallet Creation:
+
+- Fund this wallet address on L1
+- `kubectl exec <VALIDATOR_POD_NAME> -it -- /usr/local/bin/nitro --metrics --conf.file=/config/config.json --node.validator.use-smart-contract-wallet --node.validator.only-create-wallet-contract --l1.wallet.password=<SECURE_PASSWORD>`
+    - This may need to be rerun if you receive a context deadline exceeded error
+- Provide this last address to Offchain Labs for whitelisting
```

Comment:
(Same as below): clarify address doesn't need to be whitelisted for watchtower mode